commands/deploy.go

```diff
@@ -132,10 +149,17 @@ func init() {
 		"template-file",
 		"t",
 		"",
-		"Path to the CloudFormation template to be deployed",
+		"Path to the CloudFormation template to be deployed.  Must not specify template-url.",
```

Comment:
Could we perhaps reword this to the following? 
```suggestion
		"Path to the CloudFormation template to be deployed (This argument conflicts with \"template-url\")",
```